tools/ci/format_changelog

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+
```

Comment:
```suggestion
#!/usr/bin/env bash

set -euo pipefail
```

Reply:
hmm @chainchad it seems like if we set this we will get this:
```
./tools/ci/format_changelog: line 114: GITHUB_OUTPUT: unbound variable
```
which exits the script

Reply:
This means it's working :)

What I would do is check for all of these vars that should be set at the top and exit if they are not set like:

```
if [[ -z "${GITHUB_OUTPUT:-}" ]]; then
  echo "GITHUB_OUTPUT environment variable is not set."
  exit 1
fi
```